docs/source/common/optimizers.rst

```diff
@@ -21,10 +21,21 @@ Manual optimization
 For advanced research topics like reinforcement learning, sparse coding, or GAN research, it may be desirable
 to manually manage the optimization process. To do so, do the following:
 
-* Override your LightningModule ``automatic_optimization`` property to return ``False``
+* Set the ``automatic_optimization`` property to ``False`` in your ``LightningModule`` ``__init__`` function
 * Drop or ignore the optimizer_idx argument
 * Use ``self.manual_backward(loss)`` instead of ``loss.backward()``.
 
+.. testcode:: python
```

Comment:
Nice addition!

I don't know if it's doable, but putting that code snippet below the right bullet might improve understandability

Reply:
Good idea !